site/en/docs/extensions/mv3/xhr/index.md

```diff
@@ -86,56 +79,31 @@ When using resources retrieved via `XMLHttpRequest`, your background page should
 fall victim to [cross-site scripting][9]. Specifically, avoid using dangerous APIs such as the
```

Comment:
I can't comment but on the line above we should replace the mention of `XMLHttpRequest`.